### DISCUSSION TITLE
There is no extension able to load the configuration for "pimcore_ecommerce_framework"

### DISCUSSION THREAD

**Discussion by SamyMP** [pimcore/pimcore#13496] [Q&A / Support] [answered]
Hi,

I am trying to contribute to pimcore and when installing the demo project and trying to link my forked pimcore with the demo project, I run into this error : 
![image](https://user-images.githubusercontent.com/35225430/199470621-6c4fad25-1d6b-4935-8f91-d900fab9e98d.png)

Can someone help me understand why it happens and how to solve this problem ?

Thank you

**Comment by jdreesen** [comment, marked as answer]
Sounds like the ecommerce framework bundle is not enabled.

**Reply by SamyMP** [reply]
How can I enable it ? I can't use `bin/console` commands since it throws this error too.

**Reply by jdreesen** [reply]
Add it to the `config/bundles.php`

**Reply by SamyMP** [reply]
I copied the content of this file https://github.com/pimcore/demo/blob/11.x/config/bundles.php into the `bundles.php` and it seems to work, thank you !
